packages/vite/src/node/build.ts

```diff
@@ -592,6 +592,10 @@ export function resolveLibFilename(
     extension = format === 'es' ? 'mjs' : 'js'
   }
 
+  if (format === 'cjs' || format === 'es') {
```

Comment:
Not for this PR, but should we allow `es`(deprecate this) / `esm` (new value)

Reply:
Do you have some links to support `esm` over `es`? Rollup is using `es` for their format config, I think alignment with them is good here.

Reply:
Oh interesting, I thought this was coming from cjs/esm wordings that also e.g. TypeScript is using